docker-image/v1.11/arm64/debian-azureblob/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV QEMU_DOWNLOAD_SHA256 a1ef52971537e11915565233f48aa179839f676008d7911c05b3ae9
 RUN apk add curl --no-cache
 RUN curl -sL -o qemu-3.0.0+resin-aarch64.tar.gz https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-aarch64.tar.gz && echo "$QEMU_DOWNLOAD_SHA256 *qemu-3.0.0+resin-aarch64.tar.gz" | sha256sum -c - | tar zxvf qemu-3.0.0+resin-aarch64.tar.gz -C . && mv qemu-3.0.0+resin-aarch64/qemu-aarch64-static .
 
-FROM fluent/fluentd:v1.11.3-debian-arm64-1.1
+FROM fluent/fluentd:v1.11.4-debian-arm64-1.1
```

Comment:
Off topic: I'd bumped suffix version to `1.1` for applying debian slim image for armv7 and arm64:
* https://github.com/fluent/fluentd-docker-image/commit/abc3dcf0d12fc6087c1fe5e01eb1a39ca24799a6#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52
* https://github.com/fluent/fluentd-docker-image/commit/555f1eb2e8ea12dffd32a24853b6911da6385403#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

We should keep as-is for now....

Reply:
I will use `1.0` for v1.11.4 docker image.